packages/commonjs/README.md

```diff
@@ -174,6 +174,13 @@ If you set `esmExternals` to `true`, this plugins assumes that all external depe
 
 You can also supply an array of ids to be treated as ES modules, or a function that will be passed each external id to determine if it is an ES module.
 
+### `nodeDefaultImport`
```

Comment:
Thanks for moving forward here!

I was thinking about the name of the option and I am not happy about this choice as it references an environment (Node) that may change while IMO it should rather reflect functionality. So what this is about is as I understand it is to skip the default import interop in favour of assuming the default is `module.exports`. There are several ways to handle this (names are just suggestions):

* `moduleExportsIsDefault: true | false | "auto"` where "auto" is the current behaviour (could also be "detect")
* `cjsDefaultExport: "module.exports" | "default" | "auto"` where "auto" is the current behaviour (or `cjsDefaultImport` depending which way you view it)

Admittedly that the version where the default export is always `exports.default` may not be as useful, but what do I know. Could be a micro-optimisation for some. Thoughts?

Reply:
Not sure we should change the default, but I keep twisting the idea of a config plugin in my head that would just configure commonjs, node-resolve and rollup for Node compatibility.

Reply:
I have a _slight_ preference for the `true | false | "auto"` option, since typing `"module.exports"` inside a string (thus often without autocompletion) can be a bit harder.

There are already precedents for `boolean | string` options, such as [`requireReturnsDefault`](https://github.com/rollup/plugins/blob/master/packages/commonjs/README.md#requirereturnsdefault).

> Admittedly that the version where the default export is always exports.default may not be as useful, but what do I know. Could be a micro-optimisation for some. Thoughts?

We actually have an option in Babel exactly for that (https://babeljs.io/docs/en/babel-plugin-transform-modules-commonjs#nointerop), even if we don't have data about how many people use it.

Reply:
But you do not have the `false` case, i.e. always assuming the default export is `module.exports.default`, right? That's the one where I am not sure people really would use it (and if we should even add it).

Reply:
`noInterop: true` means "always get `.default`, without first checking if it has `__esModule`: https://github.com/babel/babel/tree/main/packages/babel-plugin-transform-modules-commonjs/test/fixtures/noInterop/import-default-only

Reply:
Ah I see. So reading babel/babel#12838 I understand now that you are also going for a three-state flag 👍

> or defaultIsModuleExports

I would be fine with that as well.